content/Product/OpenshiftMonitoring/faq.md

```diff
@@ -0,0 +1,140 @@
+# Debugging the in-cluster monitoring stack
+
+## Overview
+
+This serves as a collection of resources that relate to FAQ around configuring/debugging the in-cluster monitoring stack.
+
+## Identify the cause of some common issues
+
+### ServiceMonitor created but no metrics from the service
+
+The monitoring stack relies on the `ServiceMonitor` custom resource in order to tell Prometheus what endpoints to scrape.
+
+A detailed description of how the resources are linked exists [here](https://prometheus-operator.dev/docs/operator/troubleshooting/#troubleshooting-servicemonitor-changes), but we will walk through some common issues to debug the case of missing metrics.
+
+1. Ensure the `serviceMonitorSelector` in the `Prometheus` CR matches the key in the `ServiceMonitor` labels.
+2. The `Service` you want to scrape *must* have an explicitly named port.
+3. The `ServiceMonitor` *must* reference the `port` by this name.
+
+Assuming this criteria is met but the metrics don't exist, we can try debug the cause.
+
+There is a possibility Prometheus has not loaded the configuration yet. The following metrics will help to determine if that is in fact the case or if there are errors in the configuration:
+
+```bash
+prometheus_config_last_reload_success_timestamp_seconds
+prometheus_config_last_reload_successful
+```
+
+If there are errors with reloading the config, it is likely invalid and examining the logs will highlight this.
+
+Assuming that the reload was a success then the Prometheus should see the configuration.
+
+```bash
+oc exec -n openshift-monitoring prometheus-k8s-0 -c prometheus -- curl http://localhost:9090/api/v1/status/config | grep "<service-monitor-name>"
+```
+
+If the `ServiceMonitor` does not exist in the output, the next step would be to investigate the logs of both `prometheus` and the `prometheus-operator` for errors.
```

Comment:
Which prometheus? there might be many (2 reps for PM and 2 reps for UWM at least)

Reply:
now covered with a new note: ```The examples below show the namespace `openshift-monitoring`, which can be replaced with `openshift-user-workload-monitoring` when dealing with `UWM`.```

Reply:
👍🏽